crates/e2e/tests/e2e/eth_flow.rs

```diff
@@ -83,14 +84,34 @@ async fn local_node_eth_flow_indexing_after_refund() {
 
 async fn eth_flow_tx(web3: Web3) {
     shared::tracing::initialize_reentrant(
-        "warn,orderbook=debug,orderbook=debug,solver=debug,autopilot=debug",
+        "e2e=debug,orderbook=debug,solver=debug,autopilot=debug,\
+         orderbook::api::request_summary=off",
     );
     shared::exit_process_on_panic::set_panic_hook();
+
+    crate::services::clear_database().await;
     let contracts = crate::deploy::deploy(&web3).await.expect("deploy");
 
-    let mut accounts = AccountAssigner::new(&web3).await;
```

Comment:
The `AccountAssigner` seemed useful for making the test setup more succinct - just curious as to the rationale for removing it.

Reply:
I want to consistently use private key accounts everywhere. Some tests were using node/local accounts which made them harder to debug. Some components in our code can't use node accounts. The only time I use the node local account is to set up initial on chain state and fund other accounts. I should remove the whole abstraction.

Reply:
I'm going to add a commit that deletes it soon. I could see adding the component back but this time using private keys internally rather than node accounts.